src/python/alias.cpp

```diff
@@ -178,17 +210,24 @@ NB_MODULE(mitsuba_alias, m) {
     /// Only used for variant-specific attributes e.g. mi.scalar_rgb.T
     m.def("__getattr__", [](nb::handle key) { return get_attr(key); });
 
+    // `mitsuba.detail` submodule
+    nb::module_ mi_detail = m.def_submodule("detail");
```

Comment:
@wjakob At first I tried to `nb::module_::import_("mitsuba.detail");` but there was no such module, please let me know if this is not the proper way.

Reply:
That looks good to me!